community/main.py

```diff
@@ -149,6 +150,7 @@ def __init__(self):
 
 	def run(self): 
 		pyglet.app.run()
+		pyglet.app.exit()
```

Comment:
Is this necessary?

Reply:
Kinda, since I had a bug where the music would continue playing when I closed the window

Reply:
Alright, maybe a comment to explain that?

Reply:
> Kinda, since I had a bug where the music would continue playing when I closed the window

why would you not want that :D

Reply:
Solid point